database/migrations/2021_03_21_104718_force_cron_month_field_to_have_value_if_missing.php

```diff
@@ -13,7 +13,7 @@
     public function up(): void
     {
         Schema::table('schedules', function (Blueprint $table) {
-            DB::update("UPDATE `schedules` SET `cron_month` = '*' WHERE `cron_month` = ''");
+            DB::update("UPDATE schedules SET cron_month = '*' WHERE cron_month = ''");
```

Comment:
Don't use raw statements
```suggestion
            DB::table('schedules')->where('cron_month', '')->update(['cron_month' => '*']);
```